src/meteors/utils/utils.py

```diff
@@ -134,16 +134,13 @@ def agg_segmentation_postprocessing(
             If 0, use unique values in the output. Defaults to 0.
         class_axis (int, optional): The axis of the model output tensor that contains the class predictions
             if the model output is soft labels. Defaults to 1.
-        use_mask (bool, optional): Whether to use the mask to multiply the output before aggregation.
-            Defaults to True.
 
     Returns:
-        Callable[[torch.Tensor, torch.Tensor], torch.Tensor]:
-            The postprocessing function that accepts model outputs with lime mask of masked region
-            and returns aggregated outputs.
+        Callable[[torch.Tensor], torch.Tensor]: The postprocessing function that accepts model outputs with
+            lime mask of masked region and returns aggregated outputs.
```

Comment:
Yep, it caused too much trouble, and I don't even know if it was needed in the first place. If it masked input with the baseline, it would be informative for the user to know if there is any attribution in baseline values. Thus, decided not to use the mask in postprocessing.